framework/src/userobjects/ElementSubdomainModifier.C

```diff
@@ -101,9 +196,16 @@ ElementSubdomainModifier::initialize()
 {
   _moved_elems.clear();
   _moved_displaced_elems.clear();
-  _moved_nodes.clear();
+
+  _activated_elems.clear();
+  _activated_nodes.clear();
+  _activated_bnd_nodes.clear();
```

Comment:
I recall that initially we named those `activated` instead of `moved` elements/nodes, etc. However, since the ElementSubdomainModifier supports not only element activation but also element deletion/deactivation, we agreed to go with `moved` elements/nodes. Would you mind clarifying why we are switching back?